ext/standard/tests/gh10885_2.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+GH-10885: stream_socket_server context leaks
+--ENV--
+AUTO_CLOSE_RESOURCE_LIST=0
+--FILE--
+<?php
+$context = stream_context_create();
+$result = @\stream_socket_server(
+    'tcp://nonexistent:0',
```

Comment:
no point for this test really...

Reply:
Added it previously because my initial fix was wrong for the false case, but I'm ok with removing it.